### PR TITLE
Feature/random voted talks

### PR DESCRIPTION
**Pull request by svenreimers**
---
**^Add meaningful description above**

By opening a pull request you confirm that, unless explicitly stated otherwise, the changes -

 - are all your own work, and you have the right to contribute them.
 - are contributed solely under the terms and conditions of the Apache License 2.0 (see section 5 of the license for more information).

Please make sure (eg. `git log`) that all commits have a valid name and email address for you in the Author field.

If you're a first time contributor, see the Contributing guidelines for more information.

### CODE REVIEW

File: cinema/src/main/resources/tweetwall.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License (MIT)
  *
- * Copyright (c) 2019 TweetWallFX
+ * Copyright (c) 2013 TweetWallFX
```
Comment (reinhapa) [inline cinema/src/main/resources/tweetwall.css]: I guess you meant 2023 here 😎

File: exhibition/src/main/resources/tweetwall.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License (MIT)
  *
- * Copyright (c) 2019 TweetWallFX
+ * Copyright (c) 2013 TweetWallFX
```
Comment (reinhapa) [inline exhibition/src/main/resources/tweetwall.css]: Still wrong year 😎 (blast from the past)

File: hallway/src/main/resources/tweetwall.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License (MIT)
  *
- * Copyright (c) 2019 TweetWallFX
+ * Copyright (c) 2013 TweetWallFX
```
Comment (reinhapa) [inline hallway/src/main/resources/tweetwall.css]: 😎

**Review by reinhapa — DISMISSED** [review]
Back to the future with the copyright year 😎

**Review by mklaehn — DISMISSED** [review]

**Review by mklaehn — APPROVED** [review]

**Pull request merged**